atest/testdata/tags/include_and_exclude.robot

```diff
@@ -14,6 +14,10 @@ Incl-123
     [Tags]  incl_1  incl_2  incl_3
     No Operation
 
+Robot-exclude
+    [Tags]  robot:exclude
+    Fail  test not executed
+
```

Comment:
This could be at the end of the suite. There could also be two tests using the tag in different variations like `robot:exclude` and `ROBOT: EXCLUDE`.